packages/cdktf/lib/terraform-stack.ts

```diff
@@ -333,7 +333,7 @@ export class TerraformStack extends Construct {
   /**
    * Run all validations on the stack.
    */
-  public validate() {
+  public runChildValidations() {
```

Comment:
why not:
```suggestion
  public runAllValidations() {
```
that would fit the comment of the method 😁